contracts/scripts/deploy.sh

```diff
@@ -194,8 +192,10 @@ cargo contract build --release
 
 cd "$CONTRACTS_PATH"/access_control
 
-CONTRACT=$(cargo contract instantiate --url "$NODE" --constructor new --suri "$AUTHORITY_SEED")
-ACCESS_CONTROL=$(echo "$CONTRACT" | grep Contract | tail -1 | cut -c 15-)
+ACCESS_CONTROL_CODE_HASH=$(cargo contract upload --url "$NODE" --suri "$AUTHORITY_SEED")
+ACCESS_CONTROL_CODE_HASH=$(echo "$ACCESS_CONTROL_CODE_HASH" | grep hash | tail -1 | cut -c 15-)
+ACCESS_CONTROL=$(cargo contract instantiate --url "$NODE" --constructor new --suri "$AUTHORITY_SEED")
+ACCESS_CONTROL=$(echo "$ACCESS_CONTROL" | grep Contract | tail -1 | cut -c 15-)
```

Comment:
Those `cut`s after `N` columns tends to break frequently as we depend on format of some external output command like `cargo contract` in this example, but I guess we don't have a better way for now.